### ISSUE TITLE
.clone() returns a non-clone after item is updated from server

### ISSUE THREAD

**Issue by emmanuelgeoffray** [marshallswain/feathers-pinia#52]
Here is a sample code to reproduce the error :

```ts
// init
const postStore = usePosts()
const { item: post, isPending } = useGet({ model: postStore.Model, id: props.id })

// clone
const postClone = post.clone()
postClone.name = 'new name'
postClone.commit() // WORKS

// -------- post changed from server ------------------

// clone again
postClone2 = post.clone()
postClone.name = 'other name'
postClone2.commit() // ERROR 'You cannot call commit on a non-copy'
```

To me, it seems to be an issue with the existing clone that gets updated with a `__isClone: false` when an updated item is received from server.

Here is a quick fix 

```ts
// clone
const postClone = post.clone()
postClone.name = 'new name'
postClone.commit()

// -------- post changed from server ------------------

// reset existing clone
postStore.clonesById[props.id] = null // HOTFIX

postClone2 = post.clone()
postClone.name = 'other name'
postClone2.commit() // WORKS
```

**Comment by marshallswain** [comment]
Thanks for reporting.

**Comment by marshallswain** [comment]
This is fixed in feathers-pinia@0.36.0
